### PR TITLE
Filling in layout

### PR DESCRIPTION
**Pull request by tranxuanthang**
- Using HTML5, SASS and bootstrap to make page layouts.
- Rewrite routes URLs & use named routes on `link_to`.
- Add contact & sign up pages.

### CODE REVIEW

**Comment by tranxuanthang** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -60,3 +60,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# Use bootstrap-sass
```
Comment (huongnguyenmta) [inline Gemfile]: remove comment

File: app/assets/javascripts/users.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (huongnguyenmta) [inline app/assets/javascripts/users.coffee]: remove file

File: app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  text-align: center;
+}
+
+h2 {
+  font-size: 1.2em;
+  letter-spacing: -1px;
+  margin-bottom: 30px;
+  text-align: center;
+  font-weight: normal;
+  color: #777;
```
Comment (huongnguyenmta) [inline app/assets/stylesheets/custom.scss]: define variable

**Comment by tranxuanthang** [comment]
ready

File: app/assets/stylesheets/users.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (huongnguyenmta) [inline app/assets/stylesheets/users.scss]: xóa file này đi em

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("title"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("home"),   root_path %></li>
```
Comment (huongnguyenmta) [inline app/views/layouts/_header.html.erb]: xóa khoảng trắng thừa

File: app/views/static_pages/home.html.erb [hunk 6]
```diff
@@ -1,7 +1,12 @@
...
-<p>
-  This is the home page for the
-  <a href="https://www.railstutorial.org/">Ruby on Rails Tutorial</a>
-  sample application.
-</p>
+<div class="center jumbotron">
+  <h1><%= t ".title" %></h1>
+
+  <h2>
+      <%= t ".introduction_html" %>
```
Comment (huongnguyenmta) [inline app/views/static_pages/home.html.erb]: indent

File: app/views/static_pages/home.html.erb [hunk 7]
```diff
@@ -1,7 +1,12 @@
...
+  </h2>
+
+  <%= link_to t(".sign_up_now"), signup_path, class: "btn btn-lg btn-primary" %>
+</div>
+
+<%= link_to image_tag("rails.png", alt: t(".alt_rails_logo")),
+                      "http://rubyonrails.org/" %>
```
Comment (huongnguyenmta) [inline app/views/static_pages/home.html.erb]: indent

File: config/routes.rb [hunk 8]
```diff
@@ -1,7 +1,10 @@
...
-  get 'static_pages/about'
-	root 'application#hello'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get "users/new"
+  get "/help", to: "static_pages#help"
+  get "/about", to: "static_pages#about"
+  get "/contact", to: "static_pages#contact"
+  get "/signup", to: "users#new"
+  root "static_pages#home"
+  # For details on the DSL available within this file,
```
Comment (huongnguyenmta) [inline config/routes.rb]: xóa comment

**Comment by tranxuanthang** [comment]
Chị check lại xem ok chưa nhé

**Comment by tranxuanthang** [comment]
ready

**Pull request merged**